test/unit/algorithms/test_germselection.py

```diff
@@ -388,4 +429,17 @@ def lite_germ_selection_end_to_end_test(self):
         
     def robust_germ_selection_end_to_end_test(self):
         robust_germs = germsel.find_germs(self.target_model, seed=2017)
-        #todo assert correctness
\ No newline at end of file
+        #todo assert correctness
+
+#helper function
+def count_gate(circuits, gate_name):
+    num_gate=0
+    for circuit in circuits:
+        num_gate+=circuit.str.count(gate_name)
+    return num_gate
+
+def count_ops(circuits):
+    num_ops = 0
+    for circuit in circuits:
+        num_ops+= circuit.num_gates
+    return num_ops
```

Comment:
Missing blank line at EOF

Reply:
Technically it's part of the UNIX standard: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file.